doc/docs/master/deploy-manage/manage/s3gateway/supported-operations.md

```diff
@@ -0,0 +1,456 @@
+
+     ```bash
+     s3cmd mb s3://master.test
+     ```
+
+     This command creates the `test` repository with the `master` branch.
+
+1. Verify that the S3 bucket has been successfully created:
+
+   * If you are using MinIO, type:
+
+     ```bash
+     mc ls local
+     ```
+
+     **System Response:**
+
+     ```bash
+     [2019-07-18 13:32:44 PDT]      0B master.test/
+     [2019-07-12 15:09:50 PDT]      0B master.train/
+     [2019-07-12 14:58:50 PDT]      0B master.pre_process/
+     [2019-07-12 14:58:09 PDT]      0B master.split/
+     [2019-07-12 14:58:09 PDT]      0B stats.split/
+     [2019-07-12 14:36:27 PDT]      0B master.raw_data/
+     ```
+
+   * If you are using AWS, type:
+
+     ```bash
+     aws --endpoint-url http://localhost:30600/ s3 ls
+     ```
+
+     **System Response:**
+
+     ```bash
+     2019-07-26 11:35:28 master.test
+     2019-07-12 14:58:50 master.pre_process
+     2019-07-12 14:58:09 master.split
+     2019-07-12 14:58:09 stats.split
+     2019-07-12 14:36:27 master.raw_data
+          ```
+   * If you are using S3cmd, type:
+
+     ```bash
+     s3cmd ls
+     ```
+
+     **System Response:**
+
+     ```bash
+     2019-07-26 11:35 master.test
+     2019-07-12 14:58 master.pre_process
+     2019-07-12 14:58 master.split
+     2019-07-12 14:58 stats.split
+     2019-07-12 14:36 master.raw_data
+     ```
+
+   * You can also use the `pachctl list repo` command to view the
+   list of repositories:
+
+     ```bash
+     pachctl list repo
+     ```
+
+     **System Response:**
+
+     ```bash
+     NAME               CREATED                    SIZE (MASTER)
+     test               About an hour ago          0B
+     train              6 days ago                 68.57MiB
+     pre_process        6 days ago                 1.18MiB
+     split              6 days ago                 1.019MiB
+     raw_data           6 days ago                 2.561MiB
+     ```
+
+     You should see the newly created repository in this list.
+
+### Delete an S3 Bucket
+
+You can delete an S3 bucket in Pachyderm from the AWS CLI or
+MinIO client by running the following command:
+
+* If you are using MinIO, type:
+
+  ```bash
+  mc rb local/master.test
+  ```
+
+  **System Response:**
+
+  ```bash
+  Removed `local/master.test` successfully.
+  ```
+
+* If you are using AWS, type:
+
+  ```bash
+  aws --endpoint-url http://localhost:30600/ s3 rb s3://master.test
+  ```
+
+  **System Response:**
+
+  ```bash
+  remove_bucket: master.test
+  ```
+
+* If you are using S3cmd, type:
+
+  ```bash
+  s3cmd rb s3://master.test
+  ```
+
+## Upload and Download File Objects
+
+For input repositories at the top of your DAG, you can both add files
+to and download files from the repository.
+When you add files, Pachyderm automatically overwrites the previous
+version of the file if it already exists.
+
+Uploading new files is not supported for output repositories,
+these are the repositories that are the output of a pipeline.
```

Comment:
This is no longer true with s3g sidecar instances

Reply:
removed.